### DISCUSSION TITLE
Want to know more about MacOS system.

### DISCUSSION THREAD

**Discussion by amoljagdalepucsd** [Ylianst/MeshCentral#5527] [Q&A]
Hi community , 
How to send some keystroke to remote mac system.
I am sending it with existing functionality but its not working 
for example : perform **control + command + Q** on remote system via keystroke
@si458

**Comment by si458** [comment]
if you right click the CTRL+ALT+DEL at bottom left hand side, you can click customise and make keys to send when u select them and click send
note tho: the is no command key, but i think it maps to the win key instead?

**Reply by amoljagdalepucsd** [reply]
Thank you @si458 for help.
I am trying to send this combination as well but its not working it seems.
do we need to enable remotekeymap and localkeymap feature while sending keystroke to macOS system.

**Reply by si458** [reply]
I'm not 100% sure I'm sorry
I will have a look when I get chance...

**Reply by amoljagdalepucsd** [reply]
Okay Thanks @si458
